app/models/accomplishments_member.rb

```diff
@@ -0,0 +1,4 @@
+class AccomplishmentsMember < ApplicationRecord
+	belongs_to :member
```

Comment:
This table just connects `accomplishments` to `members` and carries with it a little extra info. As a result it belongs to both `member` and `accomplishment` so that if a member or an accomplishment is deleted, it vanishes from this table simultaneously.